src/containers/stage.jsx

```diff
@@ -32,10 +33,12 @@ class Stage extends React.Component {
         this.attachRectEvents();
         this.attachMouseEvents(this.canvas);
         this.updateRect();
-        this.renderer = new Renderer(this.canvas);
-        this.props.vm.attachRenderer(this.renderer);
         this.audioEngine = new AudioEngine();
         this.props.vm.attachAudioEngine(this.audioEngine);
+        this.renderer = new Renderer(this.canvas);
+        this.props.vm.attachRenderer(this.renderer);
+        this.storage = new Storage();
+        this.props.vm.attachStorage(this.storage);
```

Comment:
The Stage seems like the right place to attach the renderer to the VM, pretty iffy on the audio engine, but seems wrong for Storage. It seems like something we might want to do in the GUI container, where we instantiate the VM?  Either that or in the app and pass the VM to the GUI with the storage already attached.